proposals/20210525-feast_example_gen.md

```diff
@@ -0,0 +1,75 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Badrul Chowdhury
+
+**Your email:** badrulchowdhury17@gmail.com
+
+**Your company/organization:** Apple
+
+**Project name:** ExampleGen for Feast Feature Store
```

Comment:
just confirm This FeatureStore Connector is a ExampleGen, right?

do you plan to inherit from querybasedExampleGen?
(beam io support it already? or you can reuse the BigQuerySource?)

Reply:
@1025KB Yes, `FeastExampleGen` will inherit from `QuerybasedExampleGen`. It will not query BigQuery directly though; instead, it will use Feast APIs to query the specified backend (with BigQuery being one of the available options).